tapgarden/planter.go

```diff
@@ -1030,8 +1197,8 @@ func (c *ChainPlanter) gardener() {
 
 				ctx, cancel := c.WithCtxQuit()
 				batches, err := listBatches(
-					ctx, c.cfg.Log, c.cfg.GenTxBuilder,
-					*listBatchesParams,
+					ctx, c.cfg.Log, c.cfg.ProofFiles,
```

Comment:
I think we can just use the _local_ universe directly here. It only needs to return the single state transition, rather than the entire file for the asset.

Reply:
I think for universe lookup we would need the assetID + group key + outpoint + original script key. And fetching the original script key from the DB is problematic (as outlined in a lower comment).